### DISCUSSION TITLE
支付宝付款成功，但没有收到激活码。

### DISCUSSION THREAD

**Discussion by kelvin1816** [getlantern/lantern-forum-cn#606] [Q&A]
升级到了最新的 lantern 7.8.8（for mac），但无法通过电子邮件登录（会员有效期到 2025 年 2 月）。我试图透过升级到专业版来实现登录，又购买了一年会员，支付宝付款成功，但没有收到激活码。请帮忙协助解决

**Comment by kelvin1816** [comment]
我的 Email：wongxw@hotmail.com

**Comment by MaxMKMax** [comment]
@kelvin1816 
您好. 好的, 我帮您提交问题. 但需要您提供我支付单据才能够帮您确认激活码.

**Comment by kelvin1816** [comment]
Mike,
        你好！
        我的支付宝付款截图如下，[cid:447128c6-20d0-46ab-a873-4e762c622449]
________________________________
发件人: Mike K ***@***.***>
发送时间: 2024年6月18日 22:20
收件人: getlantern/lantern-forum-cn ***@***.***>
抄送: kelvin1816 ***@***.***>; Mention ***@***.***>
主题: Re: [getlantern/lantern-forum-cn] 支付宝付款成功，但没有收到激活码。 (Discussion #606)


@kelvin1816<https://github.com/kelvin1816>
您好. 好的, 我帮您提交问题. 但需要您提供我支付单据才能够帮您确认激活码.

―

**Comment by MaxMKMax** [comment]
@kelvin1816 您好. 为了保护您的隐私, 请您直接在这个链接上提交您的资料就好了. 支付单据和邮箱. 
后续您可以在GitHub这个帖子和我跟进如果您的问题还没解决. 谢谢您!
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
